web/beacon-app/src/components/common/TagState/TagState.tsx

```diff
@@ -14,6 +14,7 @@ const StateMap = {
   [TOPIC_STATE.PENDING]: 'secondary',
   [TOPIC_STATE.ARCHIVED]: 'warning',
   [TOPIC_STATE.DELETTING]: 'error',
+  [TOPIC_STATE.INCOMPLETE]: 'error',
```

Comment:
This will make the bg color for the tag in the header a shade of red instead of defaulting to blue.

Reply:
It looks like it was previously shared. I've created a `PROJECT_STATE` map and also updated the statuses that could be returned after double checking the Tenant db files.